app/javascript/packages/verify-flow/index.tsx

```diff
@@ -38,7 +38,7 @@ export function VerifyFlow({ initialValues = {}, basePath, appName }: VerifyFlow
         <StepIndicatorStep title="Secure your account" status={StepStatus.CURRENT} />
       </StepIndicator>
       <Alert type="success" className="margin-bottom-4">
-          {t('idv.messages.confirm')}
```

Comment:
This will be your friend: 😄 

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode#format-on-save